guide/ops/high-availability-supplemental.md

```diff
@@ -0,0 +1,97 @@
+---
+title: High Availability (Supplemental)
+layout: website-normal
+---
+
+This document supplements the High Availability documentation available [here](http://brooklyn.apache.org/v/latest/ops/high-availability.html)
+and provides an example of how to configure a pair of Apache Brooklyn servers to run in master-standby mode with a shared NFS datastore
+
+### Prerequisites
+- Two VMs (or physical machines) have been provisioned
+- NFS or another suitable file system has been configured and is available to both VMs
+- An NFS folder has been mounted on both VMs at `/mnt/amp-persistence` and both machines can write to the folder
+
+### Launching
+To start, download the latest Apache Brooklyn release on both VMs, extract it and navigate to the root folder:
+
+{% highlight bash %}
+wget -O apache-brooklyn-0.9.0-bin.tar.gz "https://www.apache.org/dyn/closer.lua?action=download&filename=brooklyn/apache-brooklyn-0.9.0/apache-brooklyn-0.9.0-bin.tar.gz"
```

Comment:
You should use apache-brooklyn_{{site.brooklyn-version}} instead of naming a specific version here